### PR TITLE
[REL] stock_picking_labels to V11.

### PR DESCRIPTION
**Pull request by ValentinoDefelice**
[FIX] Update aeroo field
[FIX] Update README

### CODE REVIEW

File: stock_picking_labels/README.rst [hunk 1]
```diff
@@ -8,42 +16,30 @@ Stock Picking Labels
...
 
-
 Usage
 =====
 
 To use this module, you need to:
 
-#. Go to the Picking and print the report "Stock Picking Labels".
-
+#. Go to ...
```
Comment (zaoral) [inline stock_picking_labels/README.rst]: you replaced the actually useful info about usage with the generic ´#. Go to...´, please revert this change

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @ValentinoDefelice 

## Technical review

Leave you some comments inline code, but please also need to review this ones:

1. please remove stock_picking_labels/ __init__.py executable permissions.
2. rename stock_picking_labels/report/report_data.xml to report.xml
3. the migrations scripts need to be delete since they are nor used. if really required then need to change the version to 11.0.1.0.0

## Functional review

When I try to print the report is currently not working it throw me an error.

image

I leave you here the complete log error for more detail
https://gist.github.com/zaoral/f80c53563d3355b7ec5d624344c66e82

Also please, after make the fixes rebase this branch because is out-of-date and wont be able to merge until is up-to-date.

Best Regards

**Review by nicomacr — COMMENTED** [review]
@zaoral i've do some corrections in this PR, 
Check it if everything is correct.
Best Regards
